### DISCUSSION TITLE
Помогите с настройкой

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#5333] [Q&A]
Все вроде как рабдоатет. Но все запросы в статистике и в журнале показываапет как от localhost. Как сделать, чтобы какждое устройство(айпи) в отдельности  показывал? Очень прошу. 

![1](https://user-images.githubusercontent.com/74646605/211145407-dd051734-ac71-48ed-9221-4dbb94be9145.jpg)

**Comment by ammnt** [comment]
@fizsef, нужно больше вводных данных. Если логируется так, значит все запросы идут скорее всего с 127.0.0.1 на 53 порт. Как настроена сеть? Где стоит AdGuard? На роутере? Как настроены устройства и что у них в настройках DNS? И тд.

**Comment by fizsef** [comment]
AGH установлен  на  роутер Xiaomi Redmi Router AC2100 ,  OpenWrt 22.03.2 .  
Выкладываю конфиги файлов (сохранил как txt файлы) - 
1.  /etc/confg/dhcp
2. /etc/confg/AdGuardHome
3. /etc/confg/AdGuardHome.yaml
Думаю все в этих файлах, если нужно еще что то , то тоже выложу. Долго мучался, то вообще не работал , точнее веб интерфейс запускался, но статистика и журнал показывал, что ничего через AGH не проходит. 
Потом как-то удалось настроить вот как есть, все запросы идут от localHost(127.0.0.1) . 
Ранее , до обновления OpenWrt  все правильно работало, каждый клиент(устройство) отдельно отображалось в AGH  и никакого localhost там никогда не было. Мне тогда помогли настроить тоже, сам не могу понять все эти перенаправления портов и прочее. 
[dhcp.txt](https://github.com/AdguardTeam/AdGuardHome/files/10395255/dhcp.txt)
[AdGuardHome.txt](https://github.com/AdguardTeam/AdGuardHome/files/10395253/AdGuardHome.txt)
[AdGuardHome.yaml.txt](https://github.com/AdguardTeam/AdGuardHome/files/10395254/AdGuardHome.yaml.txt)

Прошу помочь разобраться и настроить правильно. Заранее благодарен!
